.envrc

```diff
@@ -1,6 +1,8 @@
-PATH_add ./bin
-export OAUTH_CLIENT_ID="paas-admin"
-export OAUTH_CLIENT_SECRET="oohilovesecrets"
-export AUTH_SERVER_URL="https://login.bosh-lite.com"
-export TOKEN_SERVER_URL="https://uaa.bosh-lite.com"
-export SKIP_TLS_VERIFICATION=true
+export PATH="./bin:$PATH"
+export CF_API_ENDPOINT="https://api.${DEPLOY_ENV}.dev.cloudpipeline.digital"
+export CF_AUTH_ENDPOINT=$(cf curl /v2/info | jq -r .authorization_endpoint)
+export CF_TOKEN_ENDPOINT=$(cf curl /v2/info | jq -r .token_endpoint)
+export CF_CLIENT_ID="paas-admin"
+export CF_CLIENT_SECRET=$(aws s3 cp s3://gds-paas-${DEPLOY_ENV}-state/cf-secrets.yml - | awk '/uaa_clients_paas_admin_secret/ { print $2 }')
```

Comment:
Should we include things that are so specific to our development environments? Do we want other people to use the app in the future?

Reply:
I have mixed feelings.

Setting up these variables for running in dev is painful. But I understand this is so specific to us that would be annoying for outsiders.

Henry created some scripts for running (to replace/augment .envrc) ... maybe we should document the "standard" way ie `CF_AUTH_ENDPOINT=xx CF_TOKEN_ENDPOINT=yy rails s` in the README for all to enjoy ... but then have some additional documentation for paas-specific-ness?

Reply:
That sounds good. I definitely appreciated having an easy way to run it against my dev environment, but I think it would be easier to understand if we provided generic instructions foremost.